SingularityService/src/main/java/com/hubspot/singularity/smtp/SingularityMailer.java

```diff
@@ -287,6 +291,10 @@ private void prepareTaskMail(Optional<SingularityTask> task, SingularityTaskId t
 
     final String body = Jade4J.render(taskTemplate, templateProperties);
 
+    if (LOG_EMAILS_FOR_DEBUG) {
+      LOG.debug("TheMail: " + body);
```

Comment:
I'd suggest logging this at the `LOG.trace()` level instead, so that people can more easily control whether or not the email should be logged